frontend/packages/topology/src/components/quick-search/QuickSearchModalBody.tsx

```diff
@@ -139,12 +145,17 @@ const QuickSearchModalBody: React.FC<QuickSearchModalBodyProps> = ({
     };
   }, [closeModal, onCancel, onEnter, selectNext, selectPrevious]);
 
+  const getModalHeight = () => {
+    let height: number = 60;
+    if (catalogItems?.length > 0) {
+      if (viewAll.length === 2) height = 471;
+      else if (viewAll.length === 1) height = 448;
```

Comment:
Not sure how you came to this height number but it could be converted to a simple formula like this - 
```suggestion
      height += 388 + viewAll.length*23;
```